Added "Updating Atom" section. (#513)
atom/flight-manual.atom.io: content/getting-started/sections/installing-atom.md

{{/linux}}

#### Updating Atom

> You should consider updating Atom periodically for the latest improvements to the software. When Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.

Reviewer: Reading this out loud, it feel like the first 2 sentences could you use a connector?  E.g.:

```suggestion
You should consider updating Atom periodically for the latest improvements to the software.  Additionally, when Atom receives hotfixes for security vulnerabilities, you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.
```

> You should consider updating Atom periodically for the latest improvements to the software. Additionally, when Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.

Reviewer: Sorry, I gave it one last read through in the context of the whole page and I think we don't need this sentence:

```suggestion
You should consider updating Atom periodically for the latest improvements to the software. Additionally, when Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible.
```

I think it's leftover from when this was initially proposed as a separate page.

Once we remove that I'll go ahead and merge.

Author: @rsese I agree, that just sounded awkward from a pig picture standpoint. I removed that sentence.

{{#mac}}

> Atom's default setting under the `Atom > Settings` menu will check for updates automatically, but if you don't have "Automatically Update" enabled, you can also update Atom manually.

Reviewer: Sorry I missed this section in my last review - I think it feels a little ambiguous what setting is referred to at first in "Atom's default setting under the `Atom > Settings` menu"?  The way settings are described in https://flight-manual.atom.io/getting-started/sections/atom-basics/ might be helpful, maybe something like:

```suggestion
"Automatically Update" in Core Settings of the [Settings View](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences) is enabled by default so Atom will check for updates automatically.  But if you disable this setting, you can update Atom manually.
```

To perform a manual update:

* Click on the `Atom > Check for Update` menu item in the menu bar.
* Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button.

Atom will begin to update if an update is available.

{{/mac}}

{{#windows}}

> Atom's default setting under the `File > Settings` menu will check for updates automatically, but if you don't have "Automatically Update" enabled, you can also update Atom manually.

Reviewer: Same suggestion as above:

```suggestion
"Automatically Update" in Core Settings of the [Settings View](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences) is enabled by default so Atom will check for updates automatically.  But if you disable this setting, you can update Atom manually.
```

{{#mac}}

#### Updating Atom on Mac

Atom's default setting under the `Atom > Settings` menu will check for updates automatically, but if you don't have "Automatically Update" enabled, you can also update Atom manually.

To perform a manual update:

* Click on the `Atom > Check for Update` menu item in the menu bar.
* Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button.

Atom will begin to update if an update is available.

{{/mac}}

{{#windows}}

#### Updating Atom in Windows

Atom's default setting under the `File > Settings` menu will check for updates automatically, but if you don't have "Automatically Update" enabled, you can also update Atom manually.

To perform a manual update:

* Click on the `Help > Check for Update` menu item in the menu bar.
* Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button.

Atom will begin to update if an update is available.

{{/windows}}

{{#linux}}

#### Updating Atom in Linux

> If you are not using Atom's official package repositories, Atom must be updated manually in Linux. You can download the latest version of Atom from https://atom.io  

Reviewer: I think it could be helpful to be a bit more explicit here:

> If you are not using Atom's official package repositories, you will need to manually download and install the latest .rpm or .deb package from https://atom.io. For more details, see [Installing Atom on Linux](https://flight-manual.atom.io/getting-started/sections/installing-atom/#installing-atom-on-linux).
